docs/source/start.rst

```diff
@@ -48,7 +48,7 @@ The Forest 2.0 Downloadable SDK Preview currently contains:
 The QVM and the compiler are packed as program binaries that are accessed through the command line. Both of them provide
 support for direct command-line interaction, as well as a server mode. The :ref:`server mode <server>` is required for use with pyQuil.
 
-`Request the Forest SDK here <http://rigetti.com/forest>`__. You'll receive an email right away with the download links
+`Download the Forest SDK here <https://qcs.rigetti.com/sdk-downloads>`__, where you can find links
 for macOS, Linux (.deb), Linux (.rpm), and Linux (bare-bones).
```

Comment:
can you add Windows here too while we're at it

Reply:
good call!